tests/CommonTests.cs

```diff
@@ -411,6 +411,11 @@ public async Task CanHandleKeysend()
                     case EclairLightningClient _:
                         var response = await src.Pay(param);
                         Assert.Equal(PayResult.Ok, response.Result);
+                        Assert.Null(response.ErrorDetail);
+                        Assert.NotNull(response.Details.PaymentHash);
+                        
+                        var invoice = await dest.GetInvoice(response.Details.PaymentHash);
```

Comment:
Can you check if it correspond to the pre-image? it would check that we aren't making error based on endianness.